tests/torch/sparsity/movement/test_model_saving.py

```diff
@@ -268,9 +268,8 @@ def test_ngraph_pruning(self, tmp_path: Path, desc: dict):
         pruned_file_bytes = pruned_file.stat().st_size
         not_pruned_file_bytes = not_pruned_file.stat().st_size
         file_size_ratio = 1 - pruned_file_bytes / not_pruned_file_bytes
-        assert pytest.approx(compression_rate, abs=1e-2) == desc.nncf_weight_ratio
```

Comment:
Why not increase the tolerance here?

Reply:
this line should not be removed, have revert it